extensions/arc/package.json

```diff
@@ -295,9 +295,10 @@
                           "type": "text",
                           "required": true,
                           "defaultValue": "",
-                          "enabled": {
-                            "target": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_CONNECTIVITY_MODE",
-                            "value": "direct"
+                          "enabled": false,
+                          "valueProvider": {
+                            "providerId": "subscription-id-to-tenant-id",
```

Comment:
missed updating this from providerId to id as well?

Reply:
So actually this is different and I think I want to leave it as providerId. The other ones were the actual provider implementations which make sense that they just need to say id. 

But this is the field itself defining that it wants to use a value provider with the id specified - and since it isn't actually implementing anything I think it makes sense to leave it as providerId to be very clear that this is the id of the provider and not anything else.